webcompat/static/js/lib/bugform.js

```diff
@@ -417,6 +444,8 @@ function BugForm() {
     var reader = new FileReader();
     reader.onload = _.bind(function(event) {
       var dataURI = event.target.result;
+      // storing URI to use for Image Editor background
+      that.dataURI = dataURI;
```

Comment:
ended up using `img.src = this.previewEl.get(0).style.backgroundImage.slice(5, -2);` in the code to display the Image Editor modal. let me know if I still need to do clean up on the long value tho...